app/views/comments/_single_comment.html.erb

```diff
@@ -16,7 +16,7 @@
     <% else %>
       <%# FRONT END, update this if a.user comes in %>
       <h4 class="heading byline">
-        <% cache("#{[single_comment, single_comment.comment_owner]}_heading", expires_in: 1.week) do %>
+        <% cache(["heading", single_comment, single_comment.comment_owner], expires_in: 1.week) do %>
```

Comment:
Could we put the part of the comment at the end of the key, so it looks more like
```
add ao3-v2-dev:views/comments/_single_comment:14612b2c65cb79fdb6d277d59bd6c7a3/comments/279-20231221072216000000/body 1 1708945243 319
```
(not actually generated locally so the format may be slightly different)?

Reply:
That results in a double `//` in the key for the guest comments:
```
views/comments/_single_comment:3cad2f848407f07d9e24d71e8602c075/comments/290-20240104112718000000//body
```

Reply:
I think that's OK since the first key in that second list has a double `//` in it. I know we don't often look at the cache keys manually, but from an organisational perspective it makes more sense to me to have the model name before the more specific information

Reply:
Okay, pushed :+1: